puppet/modules/profiles/manifests/monitoring/client.pp

```diff
@@ -0,0 +1,32 @@
+class profiles::monitoring::client (
+  String[1] $prometheus_username,
+  String[1] $prometheus_password,
+  Stdlib::HTTPUrl $prometheus_url = 'https://prometheus-prod-01-eu-west-0.grafana.net/api/prom/push',
+  Array[Hash] $blackbox_targets = [],
+) {
+  if $facts['os']['family'] == 'RedHat' {
+    yumrepo { 'copr:copr.fedorainfracloud.org:evgeni:node-exporter-textfile-collector-scripts':
+      descr    => 'Copr repo for node-exporter-textfile-collector-scripts owned by evgeni',
+      baseurl  => 'https://download.copr.fedorainfracloud.org/results/evgeni/node-exporter-textfile-collector-scripts/centos-stream-$releasever-$basearch/',
+      gpgcheck => '1',
+      gpgkey   => 'https://download.copr.fedorainfracloud.org/results/evgeni/node-exporter-textfile-collector-scripts/pubkey.gpg',
+      enabled  => '1',
+    }
+    stdlib::ensure_packages(['moreutils', 'node-exporter-textfile-collector-scripts'])
```

Comment:
The Debian package ships a systemd timer for the exporter, should the RPM package do the same, or should we add the timer here?

Reply:
added timer to package